src/sentry/receivers/releases.py

```diff
@@ -71,6 +71,7 @@ def resolved_in_commit(instance, created, **kwargs):
                                 'commit': instance.id,
                             }
                         )
+                    GroupAssignee.objects.assign(group=group, assigned_to=user_list[0])
```

Comment:
We're using the `find_users` method on CommitAuthor (https://github.com/getsentry/sentry/blob/master/src/sentry/models/commitauthor.py#L26) so it's possible that if multiple users in an organization have a verified email that matches the CommitAuthor email, we'll get multiple. I thought about first trying to assign if a user has the email as their primary and then falling back on `find_users`.